packages/core/src/extension/iframe-resizer-setup.ts

```diff
@@ -0,0 +1,37 @@
+export interface IframeResizerWindow extends Window {
+    parentIFrame: {
+        autoResize: (isEnabled: boolean) => void;
+        setHeightCalculationMethod: (id: string) => void;
+        size: (height: number) => void;
+    };
+}
+
+export function isIframeResizerWindow(window: Window): window is IframeResizerWindow {
+    return 'parentIFrame' in window;
+}
+
+export function iframeResizerSetup(
+    taggedElementId: string | undefined,
+    fixedHeight: number | undefined,
+): void {
+    if (isIframeResizerWindow(window)) {
```

Comment:
```suggestion
    if (!isIframeResizerWindow(window)) {
        return;
    }
```

Or if it's expected that `window` is an iframe resizer window, throw an exception here so it doesn't fail silently.

Reply:
I've thrown an error because it's anticipated to be an iframe resizer window.